### DISCUSSION TITLE
Publishing the "allure-combine" package on anaconda channels

### DISCUSSION THREAD

**Discussion by amank2022** [MihanEntalpo/allure-single-html-file#29] [Q&A]
Hi @MihanEntalpo,

I was trying to install allure-combine package using **conda install** command **instead of pip install**, but found that it is not available on https://anaconda.org/. Is it possible for you to make it available on public conda channels?

**Comment by MihanEntalpo** [comment]
Hello. Actually, you don't need my package anymore, because allure already supports generating single file report out of the box.
Just call allure with argument --single-file <filename> and it'll create you a single file report
